caniusepython3/__main__.py

```diff
@@ -97,7 +97,16 @@ def projects_from_cli(args):
 def message(blockers):
     """Create a sequence of key messages based on what is blocking."""
     if not blockers:
-        return ['You have 0 projects blocking you from using Python 3!']
+        encoding = getattr(sys.stdout, 'encoding', '')
+        if encoding:
+            encoding = encoding.lower()
+        if encoding == 'utf-8':
+            # raised hands
+            flair = b'\xf0\x9f\x99\x8c'.decode('utf-8') + '  '
```

Comment:
Can you change this to the `\u` escape so you don't have to bother with the bytes + decode step?